[Site Bug] Issue with the page: /self-hosted/latest/install/installation-source/

# Describe the bug

When trying to build timescale from source with an installation of Postgres 12, I get the following error, stating that only Postgres 13, 14 or 15 are supported:

```
HEAD is now at c93faddbe Release 2.12.1
user@pc_name:~/Git/timescaledb$ ./bootstrap
-- The C compiler identification is GNU 9.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- TimescaleDB version 2.12.1. Can be updated from version 2.12.0
-- Build type is Release
-- Install method is 'source'
-- Performing Test CC_SUPPORTS_NO_UNUSED_CLI_ARG
-- Performing Test CC_SUPPORTS_NO_UNUSED_CLI_ARG - Success
-- Performing Test CC_SUPPORTS_NO_FORMAT_TRUNCATION
-- Performing Test CC_SUPPORTS_NO_FORMAT_TRUNCATION - Success
-- Performing Test CC_STRINGOP_TRUNCATION
-- Performing Test CC_STRINGOP_TRUNCATION - Success
-- Performing Test CC_SUPPORTS_IMPLICIT_FALLTHROUGH
-- Performing Test CC_SUPPORTS_IMPLICIT_FALLTHROUGH - Success
-- Performing Test CC_SUPPORTS_NEWLINE_EOF
-- Performing Test CC_SUPPORTS_NEWLINE_EOF - Failed
-- Performing Test CC_SUPPORTS_VISIBILITY_HIDDEN
-- Performing Test CC_SUPPORTS_VISIBILITY_HIDDEN - Success
-- Using compiler GNU
-- Found Git: /usr/bin/git (found version "2.25.1") 
CMake Error at CMakeLists.txt:351 (message):
  TimescaleDB only supports PostgreSQL 13, 14 and 15

-- Configuring incomplete, errors occurred!
See also "/home/scherff/Git/timescaledb/build/CMakeFiles/CMakeOutput.log".
See also "/home/scherff/Git/timescaledb/build/CMakeFiles/CMakeError.log".
```


## What do the docs say now?

The pages for installing timescale from source state that at least an installation of Postgres 12 is required. The compatibility matrix states this as well: https://docs.timescale.com/self-hosted/latest/upgrades/upgrade-pg/

## What should the docs say?

The pages should state that at least an installation of Postgres 13 is required. The compatibility matrix should be updated. 

# Page affected

https://docs.timescale.com/self-hosted/latest/install/installation-source/
https://docs.timescale.com/self-hosted/latest/upgrades/upgrade-pg/

# Subject matter expert (SME)

-

# Screenshots

![image](https://github.com/timescale/docs/assets/84381120/529b8936-042c-44f6-bfaf-f066e4f2a9c0)

--- a/self-hosted/install/installation-source.md
+++ b/self-hosted/install/installation-source.md
@@ -15,9 +15,9 @@ or installation tool.
 
 Before you start, make sure you have installed:
 
-*   PostgreSQL 12 or later, with a development environment. For more information
-    about PostgreSQL installation, including downloads and instructions, see the
-    [PostgreSQL documentation][postgres-download].
+*   A [supported version of PostgreSQL][compatibility-matrix], with a development environment. 
+    For more information about PostgreSQL installation, including downloads and instructions, 
+    see the [PostgreSQL documentation][postgres-download].
 
 You also need:
 

--- a/self-hosted/upgrades/upgrade-pg.md
+++ b/self-hosted/upgrades/upgrade-pg.md
@@ -24,7 +24,8 @@ upgrade TimescaleDB:
 
 ||PostgreSQL&nbsp;15|PostgreSQL&nbsp;14|PostgreSQL&nbsp;13|PostgreSQL&nbsp;12|PostgreSQL&nbsp;11|PostgreSQL&nbsp;10|PostgreSQL&nbsp;9.6|
 |-|-|-|-|-|-|-|-|
-|TimescaleDB&nbsp;2.10 and higher|&#9989;|&#9989;|&#9989;|&#9989;|&#10060;|&#10060;|&#10060;|
+|TimescaleDB&nbsp;2.12 and higher|&#9989;|&#9989;|&#9989;|&#10060;|&#10060;|&#10060;|&#10060;|
+|TimescaleDB&nbsp;2.10|&#9989;|&#9989;|&#9989;|&#9989;|&#10060;|&#10060;|&#10060;|
 |TimescaleDB&nbsp;2.5 to 2.9|&#10060;|&#9989;|&#9989;|&#9989;|&#10060;|&#10060;|&#10060;|
 |TimescaleDB&nbsp;2.4|&#10060;|&#10060;|&#9989;|&#9989;|&#10060;|&#10060;|&#10060;|
 |TimescaleDB&nbsp;2.1 to 2.3|&#10060;|&#10060;|&#9989;|&#9989;|&#9989;|&#10060;|&#10060;|

